### PR TITLE
Fixed an issue with frequency selections above ~12,456

### PR DESCRIPTION
**Pull request by ghost**
It used to go to a negative value when requesting frequencies above twice the maximum, which meant it wrapped to the maximum prescaler, not the minimum. This changed only the line where the prescaler is converted into an integer, it just forces it to be above or equal to 0. If the calculated prescaler is negative, it is better to use the closest possible value (0) rather than -1. If -1 is used it is converted to 0b1111... which is interpreted as the maximum prescaler.

tl;dr: limits the prescaler to be non-negative.

### CODE REVIEW

**Comment by caternuson** [comment]
Thanks for the Pull Request, however this library is being deprecated. Please use the new replacement library here:
https://github.com/adafruit/Adafruit_CircuitPython_PCA9685
Setup instructions here:
https://learn.adafruit.com/16-channel-pwm-servo-driver/python-circuitpython

If there is still a need for the updates provided by this pull request, please resubmit based on new library:
https://github.com/adafruit/Adafruit_CircuitPython_PCA9685/pulls
